qcodes/dataset/data_set.py

```diff
@@ -936,6 +943,46 @@ def get_data_as_pandas_dataframe(self,
             dfs[name] = df
         return dfs
 
+    def write_data_to_text_file(self, path: str,
+                                   single_file: bool = False,
+                                   single_file_name: Optional[str] = None) -> None:
+        """
+        An auxiliary function to export data to a text file.
```

Comment:
It would be terrific with a (short) explanation of the format here. Especially the `single_file` case, that one gets
```
x1 y1
x2 y2
... ...
xN yN
x1 z1
x2 z2
... ...
xN zN
```
which is not obvious from the file name.

Reply:
Is that the format that we want. For that format there seems to be no limitations with lenght of the columns. 

I expected the format to be

```
x1 y1 z1
x2 y2 z2
... ...   ...
xN yN zN
```

Reply:
Yes this is the format it saves. There was a miscommunication between me and William. So no problem, I will add this info to docstring.